### DISCUSSION TITLE
uninstalled respeaker 2 and 4 mic -now cant reinstall

### DISCUSSION THREAD

**Discussion by geetee24** [OpenVoiceOS/ovos-buildroot#259] [Q&A] [answered]
uninstall both 2 and 4.  now when try to reinstall: gets:

 pip install ovos_PHAL_plugin_respeaker_4mic
Looking in indexes: https://pypi.org/simple, https://www.piwheels.org/simple
ERROR: Could not find a version that satisfies the requirement ovos_PHAL_plugin_respeaker_4mic (from versions: none)
ERROR: No matching distribution found for ovos_PHAL_plugin_respeaker_4mic
(.venv) ovos@raspOvos:~/.venv/lib/python3.11/site-packages $

**Comment by builderjer** [comment, marked as answer]
use hyphens not underscores
https://github.com/OpenVoiceOS/ovos-PHAL-plugin-respeaker-4mic/blob/0f54c45b1d7e1d461053dfd617522ee4e59a80ba/setup.py#L17
